KsApi/models/Backing.swift

```diff
@@ -9,6 +9,7 @@ public struct Backing {
   public let backerCompleted: Bool?
   public let id: Int
   public let locationId: Int?
+  public let locationName: String?
```

Comment:
Just wondering if we prefer this over just passing the `Reward` to this VC? That already has all of the shipping info and would save us a lot of code.

Reply:
Cool and what about this? 😄

Reply:
We chatted on Slack 👍

Reply:
From Slack: ` The reward contains shipping rules, but it doesn’t contain the shipping rule you chose`.